tests/main/interfaces-dbus/task.yaml

```diff
@@ -68,6 +68,13 @@ execute: |
     echo "And plug is disconnected by default"
     snap interfaces | MATCH "$DISCONNECTED_PATTERN"
 
+    # FIXME This is a pretty bad check for unsupported confinement
```

Comment:
I wonder if we should just skip the entire test on debian (avoiding this slightly ugly thing in the middle).